frontend/app/views/file_versions/_show.html.erb

```diff
@@ -23,7 +23,7 @@
               <%= render_aspace_partial :partial => "file_versions/image", :locals => {:file => file_version} %>
               <%= render_aspace_partial :partial => "file_versions/file_uri", :locals => {:file_uri => URI(file_version['file_uri'])} %>
             <% end %>
-            <%= render_aspace_partial :partial => "file_versions/file_uri", :locals => {:file_uri => uri_or_string( file_version['file_uri'] ) } %>
+            <%= render_aspace_partial :partial => "file_versions/file_uri", :locals => {:file_uri => URI( file_version['file_uri'] ) } %>
```

Comment:
`URI(aString)` will throw an exception if `aString` is a "bad URI" - I believe `file_uri` is an open text field, so we can't be sure that a valid URI is always provided.  Perhaps this could be pulled into a helper method that could handle the exception gracefully?